targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MIMXRT1170/TARGET_EVK/PinNames.h

```diff
@@ -0,0 +1,270 @@
+    GPIO_SD_B2_04 = ((4 << GPIO_PORT_SHIFT) | 13),
+    GPIO_SD_B2_05 = ((4 << GPIO_PORT_SHIFT) | 14),
+    GPIO_SD_B2_06 = ((4 << GPIO_PORT_SHIFT) | 15),
+    GPIO_SD_B2_07 = ((4 << GPIO_PORT_SHIFT) | 16),
+    GPIO_SD_B2_08 = ((4 << GPIO_PORT_SHIFT) | 17),
+    GPIO_SD_B2_09 = ((4 << GPIO_PORT_SHIFT) | 18),
+    GPIO_SD_B2_10 = ((4 << GPIO_PORT_SHIFT) | 19),
+    GPIO_SD_B2_11 = ((4 << GPIO_PORT_SHIFT) | 20),
+    GPIO_DISP_B1_00 = ((4 << GPIO_PORT_SHIFT) | 21),
+    GPIO_DISP_B1_01 = ((4 << GPIO_PORT_SHIFT) | 22),
+    GPIO_DISP_B1_02 = ((4 << GPIO_PORT_SHIFT) | 23),
+    GPIO_DISP_B1_03 = ((4 << GPIO_PORT_SHIFT) | 24),
+    GPIO_DISP_B1_04 = ((4 << GPIO_PORT_SHIFT) | 25),
+    GPIO_DISP_B1_05 = ((4 << GPIO_PORT_SHIFT) | 26),
+    GPIO_DISP_B1_06 = ((4 << GPIO_PORT_SHIFT) | 27),
+    GPIO_DISP_B1_07 = ((4 << GPIO_PORT_SHIFT) | 28),
+    GPIO_DISP_B1_08 = ((4 << GPIO_PORT_SHIFT) | 29),
+    GPIO_DISP_B1_09 = ((4 << GPIO_PORT_SHIFT) | 30),
+    GPIO_DISP_B1_10 = ((4 << GPIO_PORT_SHIFT) | 31),
+
+    GPIO_DISP_B1_11 = ((5 << GPIO_PORT_SHIFT) | 0),
+    GPIO_DISP_B2_00 = ((5 << GPIO_PORT_SHIFT) | 1),
+    GPIO_DISP_B2_01 = ((5 << GPIO_PORT_SHIFT) | 2),
+    GPIO_DISP_B2_02 = ((5 << GPIO_PORT_SHIFT) | 3),
+    GPIO_DISP_B2_03 = ((5 << GPIO_PORT_SHIFT) | 4),
+    GPIO_DISP_B2_04 = ((5 << GPIO_PORT_SHIFT) | 5),
+    GPIO_DISP_B2_05 = ((5 << GPIO_PORT_SHIFT) | 6),
+    GPIO_DISP_B2_06 = ((5 << GPIO_PORT_SHIFT) | 7),
+    GPIO_DISP_B2_07 = ((5 << GPIO_PORT_SHIFT) | 8),
+    GPIO_DISP_B2_08 = ((5 << GPIO_PORT_SHIFT) | 9),
+    GPIO_DISP_B2_09 = ((5 << GPIO_PORT_SHIFT) | 10),
+    GPIO_DISP_B2_10 = ((5 << GPIO_PORT_SHIFT) | 11),
+    GPIO_DISP_B2_11 = ((5 << GPIO_PORT_SHIFT) | 12),
+    GPIO_DISP_B2_12 = ((5 << GPIO_PORT_SHIFT) | 13),
+    GPIO_DISP_B2_13 = ((5 << GPIO_PORT_SHIFT) | 14),
+    GPIO_DISP_B2_14 = ((5 << GPIO_PORT_SHIFT) | 15),
+    GPIO_DISP_B2_15 = ((5 << GPIO_PORT_SHIFT) | 16),
+
+
+    GPIO_LPSR_00 = ((6 << GPIO_PORT_SHIFT) | 0),
+    GPIO_LPSR_01 = ((6 << GPIO_PORT_SHIFT) | 1),
+    GPIO_LPSR_02 = ((6 << GPIO_PORT_SHIFT) | 2),
+    GPIO_LPSR_03 = ((6 << GPIO_PORT_SHIFT) | 3),
+    GPIO_LPSR_04 = ((6 << GPIO_PORT_SHIFT) | 4),
+    GPIO_LPSR_05 = ((6 << GPIO_PORT_SHIFT) | 5),
+    GPIO_LPSR_06 = ((6 << GPIO_PORT_SHIFT) | 6),
+    GPIO_LPSR_07 = ((6 << GPIO_PORT_SHIFT) | 7),
+    GPIO_LPSR_08 = ((6 << GPIO_PORT_SHIFT) | 8),
+    GPIO_LPSR_09 = ((6 << GPIO_PORT_SHIFT) | 9),
+    GPIO_LPSR_10 = ((6 << GPIO_PORT_SHIFT) | 10),
+    GPIO_LPSR_11 = ((6 << GPIO_PORT_SHIFT) | 11),
+    GPIO_LPSR_12 = ((6 << GPIO_PORT_SHIFT) | 12),
+    GPIO_LPSR_13 = ((6 << GPIO_PORT_SHIFT) | 13),
+    GPIO_LPSR_14 = ((6 << GPIO_PORT_SHIFT) | 14),
+    GPIO_LPSR_15 = ((6 << GPIO_PORT_SHIFT) | 15),
+ 
+    
+    // LED_GREEN = GPIO_AD_04,
+
+    // mbed original LED naming
+    //  LED1 = LED_GREEN,
+    // LED2 = LED_GREEN,
+    // LED3 = LED_GREEN,
+    //  LED4 = LED_GREEN,
+
+    // USB Pins
+    USBTX = GPIO_AD_24,
+    USBRX = GPIO_AD_25,
```

Comment:
Please replace USBTX/USBRX with CONSOLE_TX/CONSOLE_RX 😃

Reply:
Thank you, fixed it.